lib/buildkite/test_collector/ci.rb

```diff
@@ -82,4 +83,16 @@ def circleci
       "number" => ENV["CIRCLE_BUILD_NUM"],
     }
   end
+
+  def codeship
+    {
+      "CI" => "codeship",
+      "key" => "#{ENV["CI_BUILD_ID"]}",
+      "url" => ENV["CI_PULL_REQUEST"],
+      "branch" => ENV["CI_BRANCH"],
+      "commit_sha" => ENV["CI_COMMIT_ID"],
+      "number" => nil,
```

Comment:
I had a look and it seems there's no equivalent to BUILD_NUMBER. Is it ok to have this blank? What about a `CI_TIMESTAMP`?

Reply:
TBH I'm not sure what that controls. According to the [docs](https://buildkite.com/docs/test-analytics/ci-environments#recommended-environment-variables) it's not a "recommended" environment variable. 🤷🏻